torchio/utils.py

```diff
@@ -43,12 +43,16 @@ def to_tuple(
     return value
 
 
-def get_stem(path: TypePath) -> str:
+def get_stem(
+        path: Union[TypePath, List[TypePath]]
+        ) -> Union[str, List[str]]:
     """
     '/home/user/image.nii.gz' -> 'image'
     """
-    path = Path(path)
-    return path.name.split('.')[0]
+    if isinstance(path, (str, Path)):
+        path = Path(path)
+        return path.name.split('.')[0]
+    return [Path(p).name.split('.')[0] for p in path]
```

Comment:
Is there a way to not repeat this code? Maybe leave this as it was and write a new `get_stems`? Or an inner function `_get_stem` that returns `path.name.split('.')[0]`.